conf/defaults.ini

```diff
@@ -597,6 +597,33 @@ max_attempts = 3
 # Makes it possible to enforce a minimal interval between evaluations, to reduce load on the backend
 min_interval_seconds = 1
 
+# Configures for how long alert annotations should be kept. Default is 0 which keeps them for ever
+max_annotation_age =
+
+# Configures max number of alert annotations that should be kept. Defaul value is 0 which keeps all alert annotations
+max_annotations_to_keep =
+
+#################################### Annotations #########################
+
+[annotations.dashboard]
+# Dashboard annotations means that the annotation are associated with an dashboard.
```

Comment:
Is there supposed to be an enabled/disabled option here?

Reply:
What happens if this is not enabled?

Reply:
Nothing happens if its disabled.